### ISSUE TITLE
Remove Findbugs suppression DatabaseAdapter

### ISSUE THREAD

**Issue by jverbraeken** [ericcornelissen/NervousFish#140]
- Parent issues: -
- Relevant issues: -
- Estimated effort: 1 (Joost)

* * *

## What
This issue demands to remove after #137 is merged into develop
```xml
<Match>
  <Class name="~.*\.*DatabaseAdapter" />
</Match>
```

## Why
This issue is needed for the project because #137 refractors this class, eliminating this fix

## How
This feature can be added by removing the redundant code

## Definition of ready
This features is not ready until the following issues are finished:

- Redundant suppression is removed

## Definition of done
This task can be considered done when

-
